pkg/scheduler/plugins/nodecpuamplification/least_allocated_test.go

```diff
@@ -0,0 +1,429 @@
+/*
+Copyright 2022 The Koordinator Authors.
```

Comment:
The test files are modified to utilize our own `testSharedLister` rather than the internal `cache.NewSnapshot`, and to use `NodeCPUAmplificationArgs` rather than `NodeResourcesFitArgs`, so I think it's okay to keep our own licence?